apps/marketplace/components/SellerAssessment/SellerAssessmentCriteriaStage.js

```diff
@@ -20,6 +22,33 @@ const getCriteriaNeeded = (criteriaNeeded, priceMaximum, maxDailyRate) => {
 const getCriteriaAllowed = (criteriaNeeded, priceMaximum, maxDailyRate) =>
   getCriteriaNeeded(criteriaNeeded, priceMaximum, maxDailyRate) + 2
 
+const getMessage = (domain, criteriaNeeded, essentialCriteria) => {
+  const sanitizer = dompurify.sanitize
+
+  if (domain.name === 'Platforms integration') {
+    const platformMessage = `You must submit at least ${criteriaNeeded -
+      essentialCriteria.length} <strong> &apos;Other criteria&apos;</strong>`
+
+    return <span dangerouslySetInnerHTML={{ __html: sanitizer(platformMessage) }} />
+  }
+  const message = `You must submit evidence for at least ${criteriaNeeded} ${
+    criteriaNeeded === 1 ? 'criterion' : 'criteria'
+  }.`
+  return message
+}
```

Comment:
1. The message text is a bit different to the designs.  Which version are we using?
2. Do we need to use the dompurify package?  What if we did something like this?

```javascript
  getMinimumMessage = (criteriaNeeded, essentialCriteria) => {
    if (essentialCriteria.length > 0) {
      return (
        <span>
          You must submit evidence for at least {criteriaNeeded - essentialCriteria.length}{' '}
          <strong> &apos;Other criteria&apos;</strong>
        </span>
      )
    }

    return (
      <span>
        You must submit evidence for at least {criteriaNeeded} {criteriaNeeded === 1 ? 'criterion' : 'criteria'}
      </span>
    )
  }
```